### PR TITLE
Add Private Click Measurement (defer).

### LINKED ISSUE #161: Private Click Measurement
## Request for Mozilla Position on an Emerging Web Specification

* Specification Title: Privacy Preserving Ad Click Attribution
* Specification or proposal URL: https://wicg.github.io/ad-click-attribution/index.html
* Caniuse.com URL (optional): 
* Bugzilla URL (optional): 
* Mozillians who can provide input (optional): @ehsan, @englehardt, @tanvihacks, @bakulf

### Other information
We're interested in this specification in order to support a way for a click source to measure conversions as a result of the user clicking on an ad without sharing user-identifying information with the click source in the third-party context.

### PR DESCRIPTION
**Pull request by dbaron**
Closes #161.

### CODE REVIEW

**Comment by annevk** [comment]
My apologies for not writing https://github.com/mozilla/standards-positions/issues/161#issuecomment-633888164 sooner, but I would prefer if we waited with this because of that.

**Comment by dbaron** [comment]
I've revised this to be a `defer` position; please take another look.

**Comment by dbaron** [comment]
Er, given that github says there are conflicts, I should probably rebase first.

File: activities.json [hunk 1]
```diff
@@ -638,6 +638,18 @@
...
+    "ciuName": null,
+    "description": "This specification defines a privacy preserving way to attribute a conversion, such as a purchase or a sign-up, to a previous ad click.",
+    "id": "private-click-measurement",
+    "mozBugUrl": null,
+    "mozPosition": "defer",
+    "mozPositionDetail": "We're interested in this specification in order to support a way for a click source to measure conversions as a result of the user clicking on an ad without sharing user-identifying information with the click source in the third-party context.  However, we are waiting to take a position until the fraud preventions that it depends on are specified and we can understand their properties.",
+    "mozPositionIssue": 161,
+    "org": "Proposal",
+    "title": "Private Click Measurement",
+    "url": "https://privacycg.github.io/private-click-measurement"
```
Comment (annevk) [inline activities.json:651]:
```suggestion
    "url": "https://privacycg.github.io/private-click-measurement/"
```

**Pull request merged**